controllers/users.js

```diff
@@ -58,7 +61,5 @@ module.exports.login = (req, res) => {
         .send(token)
         .end();
     })
-    .catch((err) => {
-      res.status(401).send({ message: err.message });
-    });
+    .catch(new Error401('Ошибка ввода'));
```

Comment:
Нужно исправить: ошибка не обрабатывается централизованным обработчиком.
Централизованная обработка ошибок запустится только если будет вызван `next`:
```suggestion
    .catch(next);
```

Только нужно не забыть объявить `next` здесь:
https://github.com/MaximMaslovVik/15_sprint/blob/2ad864e57b00f8564f3658374d392b861c3b10fb/controllers/users.js#L50